src/renderers/WebGLMultipleRenderTargets.js

```diff
@@ -2,9 +2,9 @@ import { WebGLRenderTarget } from './WebGLRenderTarget.js';
 
 class WebGLMultipleRenderTargets extends WebGLRenderTarget {
 
-	constructor( width, height, count ) {
+	constructor( width, height, count, options ) {
```

Comment:
Do you mind updating the documentation page by adding the new parameter?

https://threejs.org/docs/index.html#api/en/renderers/WebGLMultipleRenderTargets

Reply:
Just submitted that update.  👍 

I noticed that the old page has an open parenthesis next to the `options` list item; 
I replicated it for consistency, but I can also edit both pages to say `An option` or `option` if you'd prefer.